src/Connection/Expr.php

```diff
@@ -4,19 +4,10 @@
 
 namespace Keboola\SnowflakeDwhManager\Connection;
 
-class Expr
+interface Expr
```

Comment:
Tu bola trieda `Expr` ktora sluzi na to, ak chces do query dostat nieco bez escapovania.
No fungovala iba pre `string`, a `statement_timeout_in_seconds` musi byt `int`, inak to spadne.

Preto je tu tento interface a nove triedy `ExprString` a `ExprInt`.